mbed/mbed.py

```diff
@@ -3011,6 +3011,51 @@ def help_():
     return parser.print_help()
 
 
+@subcommand('unittest',
+    dict(name='--skip-build', action='store_true', help='skip build step'),
+    dict(name='--skip-run', action='store_true', help='skip run step'),
+    dict(name='--clean', action='store_true', help='clean build data'),
+    dict(name=['-d', '--debug'], action='store_true', help='enable debug build'),
+    dict(name='--coverage', choices=['html', 'xml', 'both'], help='generate code coverage report'),
+    dict(name=['-m', '--make-program'], choices=['gmake', 'make', 'mingw32-make', 'ninja'], help='which make program to use'),
+    dict(name=['-g', '--generator'], choices=['Unix Makefiles', 'MinGW Makefiles', 'Ninja'], help='which CMake generator to use'),
+    dict(name=['-r', '--regex'], help='run tests matching regular expression'),
+    dict(name='--build-path', help='specify build path'),
```

Comment:
Can you please rename this to just `--build` to be inline with the other `mbed` commands?